README.md

```diff
@@ -256,3 +269,6 @@ However there are 2 caveats to this caveat.
 
 The JAR files (`$SDK_ROOT/platforms/android-$VERSION/android.jar`) can be used instead. But compiled JARs do not include JavaDoc and method parameter names. This JAR is automatically included by Gradle when `android_sdk_config` >> `add_gradle_deps` is specified.
 
+## Contributing
+See the wiki for architecture-related documents.
```

Comment:
We should add a CONTRIBUTING.md (can do in a separate PR.)

For inspiration: https://github.com/dart-lang/ffigen/blob/master/CONTRIBUTING.md

The markdown file can link to the wiki.